packages/sources/cryptocompare-test/package.json

```diff
@@ -28,7 +28,7 @@
     "start": "yarn server:dist"
   },
   "dependencies": {
-    "@chainlink/external-adapter-framework": "0.12.0",
+    "@chainlink/external-adapter-framework": "0.19.0",
```

Comment:
I'm being a bit annoying, but could we please update to the latest framework version? Wanna make sure we get the updated timestamp labels. Otherwise this PR LGTM